### ISSUE TITLE
[Docs Request] How do we lint our code up until the time `deno_lint` is ready?

### ISSUE THREAD

**Issue by David-Else** [denoland/deno#6043]
I have spent a while trying to configure `@typescript-eslint/eslint-plugin` to work with Deno and having no luck. It won't pick up the types on the `Deno` namespace.

For example, the VSCode extension works and sees the types, but ESLint does not (here is the hover over):
```
(property) Deno.RunOptions.cmd: string[]
Arguments to pass. Note, the first element needs to be a path to the binary Arguments to pass. Note, the first element needs to be a path to the binary

Unsafe asignment of type any[] to a variable of type string[].eslint@typescript-eslint/no-unsafe-assignment
```
All over my code when there is a reference to `Deno` there are red squiggles and complaints about 'unsafe' references to `any`.

Is there a solution to this? If so, can we get it in the docs?

**Comment by bartlomieju** [comment]
Ref #6125

**Comment by David-Else** [comment]
@bartlomieju Thanks, but the question was '_up until the time_ `deno_lint` is ready'. Are you saying it is possible to use `deno lint` now somehow?

Is it possible to run eslint on a deno project and solve the problem I outlined above? Cheers.

**Comment by bartlomieju** [comment]
> @bartlomieju Thanks, but the question was '_up until the time_ `deno_lint` is ready'. Are you saying it is possible to use `deno lint` now somehow?

Not yet, but `deno lint` will be shipped in `1.1` in a week.

> Is it possible to run eslint on a deno project and solve the problem I outlined above? Cheers.

Unfortunately, I don't know

**Comment by nayeemrmn** [comment]
This can be closed now.
